CONTRIBUTING.md

```diff
@@ -224,32 +229,31 @@ As a project, Juju follows a specific workflow:
 
 Naturally, it is not so linear in practice. Each of these is elaborated below.
 
-Sync with upstream
-------------------
+Branches
+--------
 
-First check that the branch is on `develop`:
+Generally there are multiple versions of Juju in development concurrently,
+and so we keep a separate Git branch for each version. When submitting a
+patch, please make sure your changes are targeted to the correct branch.
 
-```bash
-git branch
-* develop
-  old_feature
-```
-
-Then pull in the latest changes from upstream, assuming you have done the setup
-as above:
+The currently active branches are:
```

Comment:
Would it be better to just give this as an example, or say "As of <date>, the active branches are:" so that we don't need to keep updating this?

Reply:
I think we should keep it updated. The whole point is so that contributors know where to target their changes. Branch changes are relatively infrequent so I don't think it's a huge issue (the next change won't need to be till 3.1 is released).